src/mbgl/gl/context.hpp

```diff
@@ -81,7 +81,10 @@ class Context : private util::noncopyable {
 
     template <RenderbufferType type>
     Renderbuffer<type> createRenderbuffer(const Size size) {
-        static_assert(type == RenderbufferType::RGBA || type == RenderbufferType::DepthStencil,
+        static_assert(type == RenderbufferType::RGBA ||
+                      type == RenderbufferType::DepthStencil ||
+                      type == RenderbufferType::DepthComponent ||
+                      type == RenderbufferType::RGBA4,  // TODO I added RGBA4 because in JS RGBA8 (here 'RGBA') emits a "renderbufferStorage: invalid internalformat" warning (though still renders fine) -- investigate
```

Comment:
For some reason, `glRenderbufferStorage` doesn't support a RGBA8 format by default. For OpenGL ES 2.0, there [is an extension](https://www.khronos.org/registry/OpenGL/extensions/OES/OES_rgb8_rgba8.txt) that seems widely supported (and probably the reason why we haven't run into this issue on mobile devices before). For WebGL, I haven't found an extension.

The issue with RGB4 is that it uses less precision (only 4 bits per channel), so we'll end up with less precise colors. An alternative might be to render to a texture rather than a Renderbuffer.